core/converters.go

```diff
@@ -96,3 +97,27 @@ func SecondsToHourMinSec(input int) string {
 
 	return result
 }
+
+// EpochStartIdentifier returns the string for the epoch start identifier
+func EpochStartIdentifier(epoch uint32) string {
+	return fmt.Sprintf("epochStartBlock_%d", epoch)
+}
+
+// IsUnknownEpochIdentifier return if the epoch identifier represents unknown epoch
+func IsUnknownEpochIdentifier(identifier []byte) (bool, error) {
+	splitString := strings.Split(string(identifier), "_")
+	if len(splitString) == 0 || len(splitString[0]) == 0 {
+		return false, ErrInvalidIdentifierForEpochStartBlockRequest
+	}
+
+	epoch, err := strconv.ParseUint(splitString[1], 10, 32)
+	if err != nil {
+		return false, ErrInvalidIdentifierForEpochStartBlockRequest
+	}
+
+	if epoch == math.MaxUint32 {
```

Comment:
Why would the epoch be set to MaxUInt32?

Reply:
when running the node for the first time, you do not know what epoch the system is in.

Reply:
ok